### PR TITLE
doc: Add LoRa Cloud & Ubidots integration guide for LR1110

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
An integration guide showing how to obtain LR1110-based tracking device's geolocation results from LoRa Cloud DAS and how to visualize that data on a map widget in Ubidots.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/integrations/lr1110-geolocation/_index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+{{< figure src="lora-cloud-integration.png" alt="LoRa Cloud integration" >}}
+
+At this point, your device will send the location updates to the LoRa Cloud™ Device & Application Services. After receiving your device's location solution from Device & Application Services, in your device's **Live data** tab in {{% tts %}}, you will find a message of the **Forward location solved message** type. This message contains the object of the following structure:
+
+```bash
+"location_solved": {
+      "service": "lora-cloud-device-management-v1-wifi",
+      "location": {
+        "latitude": 51.455274,
```
Comment (adriansmares) [inline doc/content/integrations/lr1110-geolocation/_index.md:61]: Please cut two or three digits in the `latitude` and `longitude`. This is my _exact_ home address.

**Review by adriansmares — APPROVED** [review]
Content LGTM, ~but leaving main review to @benolayinka~ .

File: doc/content/integrations/lr1110-geolocation/_index.md (hunk 1 above)
Comment (nejraselimovic) [reply doc/content/integrations/lr1110-geolocation/_index.md]: I see the issue, so I cut 3 digits and zoomed out the map on the end too

**Review by johanstokking — COMMENTED** [review]
I'm with @adriansmares here, you can merge

**Pull request merged**